src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebResponseObject.Common.cs

```diff
@@ -13,48 +13,35 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// WebResponseObject.
     /// </summary>
-    public partial class WebResponseObject
+    public class WebResponseObject
     {
         #region Properties
 
         /// <summary>
-        /// Gets or protected sets the response body content.
+        /// Gets or sets the BaseResponse property.
         /// </summary>
-        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
-        public byte[] Content { get; protected set; }
+        public HttpResponseMessage BaseResponse { get; set; }
 
         /// <summary>
-        /// Gets the response status code.
+        /// Gets or protected sets the response body content.
         /// </summary>
-        public int StatusCode
-        {
-            get { return (WebResponseHelper.GetStatusCode(BaseResponse)); }
-        }
+        [SuppressMessage("Microsoft.Performance", "CA1819:PropertiesShouldNotReturnArrays")]
+        public byte[] Content { get; protected set; }
 
         /// <summary>
-        /// Gets the response status description.
+        /// Gets the Headers property.
         /// </summary>
-        public string StatusDescription
+        public Dictionary<string, IEnumerable<string>> Headers
         {
-            get { return (WebResponseHelper.GetStatusDescription(BaseResponse)); }
-        }
+            get
```

Comment:
Can convert to expression body.

Reply:
I don't see this in new commits.

Reply:
I didn't do it yet, can you give me some advice on how to do it in the correct way?

Reply:
I added example in my previous comment.

Reply:
Thanks, done